src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
I still don't think it's helpful - education of weird aspects of an api don't really belong in random places in code unless it's extremely likely that you'll hit the problem again.

In this case, I think it's unlikely.